docs/how-to/cluster/migrate.md

```diff
@@ -52,104 +63,85 @@ Building full `sasl.jaas.config` for authorisation:
-topics=.*
-groups=.*
-
-# the expected number of concurrent MirrorMaker tasks, usually set to match number of physical cores on the target cluster
-tasks.max=3
-
-# the new replication.factor for topics produced to the target cluster
-replication.factor=2
-
-# allows new topics and groups created mid-migration, to be copied
-refresh.topics.enabled=true
-sync.group.offsets.enabled=true
-sync.topic.configs.enabled=true
-refresh.topics.interval.seconds=5
-refresh.groups.interval.seconds=5
-sync.group.offsets.interval.seconds=5
-emit.checkpoints.interval.seconds=5
-
-# filters out records from aborted transactions
-old.consumer.isolation.level=read_committed
-new.consumer.isolation.level=read_committed
-
-# Specific Connector configuration for ensuring Exactly-Once-Delivery (EOD)
-# NOTE - EOD support guarantees released with Apache Kafka 3.5.0 so some of these options may not work as expected
-old.producer.enable.idempotence=true
-new.producer.enable.idempotence=true
-old.producer.acks=all
-new.producer.acks=all
-# old.exactly.once.support = enabled
-# new.exactly.once.support = enabled
-```
-
-Once these properties file has been prepared, place it on every Apache Kafka unit:
-
-```bash
-cat mm2.properties | juju ssh kafka/<id> sudo -i 'sudo tee -a /var/snap/charmed-kafka/current/etc/kafka/mm2.properties'
-```
-
-## Starting a dedicated MirrorMaker cluster
+## Run MirrorMaker cross-cluster replication task
 
-We strongly recommend running MirrorMaker services on the downstream (target) cluster to avoid service impact due to resource use. Now that the properties are set on each unit of the new cluster, the MirrorMaker services can be started with JMX metrics exporters.
-
-Prepare the `KAFKA_OPTS` environment variable for running with an exporter:
+First, get the `admin` credentials for the Charmed Apache Kafka Connect application:
 
 ```bash
-export KAFKA_OPTS="-Djava.security.auth.login.config=/var/snap/charmed-kafka/current/etc/kafka/zookeeper-jaas.cfg -javaagent:/var/snap/charmed-kafka/current/opt/kafka/libs/jmx_prometheus_javaagent.jar=9099:/var/snap/charmed-kafka/current/etc/kafka/jmx_kafka_connect.yaml"
+CONNECT_SECRET_KEY=$(juju list-secrets | grep kafka-connect | awk '{ print $1}')
+export CONNECT_USERNAME=admin
+export CONNECT_PASSWORD=$(juju show-secret --reveal $CONNECT_SECRET_KEY --format yaml | yq '.. | ."admin-password"? // empty' | tr -d '"')
+export CONNECT_ENDPOINTS=$(juju show-unit kafka-connect/0 --format json | yq '.. | ."public-address"? // empty' | tr -d '"')
 ```
 
-Start MirrorMaker on each target Charmed Apache Kafka unit:
+To start the MirrorMaker replication task, make an HTTP request to Charmed Apache Kafka Connect, using the credentials and endpoints for both Kafka clusters:
 
 ```bash
-juju ssh kafka/<id> sudo -i 'cd /snap/charmed-kafka/current/opt/kafka/bin && KAFKA_OPTS=$KAFKA_OPTS ./connect-mirror-maker.sh /var/snap/charmed-kafka/current/etc/kafka/mm2.properties'
+curl -u $CONNECT_USERNAME:$CONNECT_PASSWORD \
```

Comment:
Question: This is a big code block. Do you think we can hide it in a collapsible block?

Reply:
Will it all render, or truncate the block with it's own scroll-bar?

Reply:
TIL about the render link, that's awesome! Yeah give the vertical scrolling an explore if you have the time, but I'll add it collapsible in the meantime anyway, thanks!